apps/www/components/ui/alert.tsx

```diff
@@ -4,7 +4,7 @@ import { VariantProps, cva } from "class-variance-authority"
 import { cn } from "@/lib/utils"
 
 const alertVariants = cva(
-  "relative w-full rounded-lg border p-4 [&>svg]:absolute [&>svg]:text-foreground [&>svg]:left-4 [&>svg]:top-4 [&>svg+div]:translate-y-[-3px] [&:has(svg)]:pl-11",
+  "relative w-full rounded-lg border p-4 [&>svg]:absolute [&>svg]:text-foreground [&>svg]:left-4 [&>svg]:top-4 [&>svg+div]:translate-y-[-3px] @if($('svg', this).length > 0) pl-11 @endif",
```

Comment:
Can you link me to the docs for this? I've never seen this in Tailwind. Thank you.

Reply:
I had never seen this either, so I decided to look for it. I couldn't find it so I was curious if it actually works and pulled the changes of this PR. Turns out it doesn't work as @shadcn originally intended, at least when I tried it.

When I went to test it without an SVG as child, the `@if($('svg', this).length > 0) ... @endif` had no effect,  the class `pl-11` will always be rendered regardless of whether there is an SVG element present as a child.

Reply:
Will this be fixed?

Reply:
> Will this be fixed?

It won't get fixed with `@if` as that isn't Tailwind syntax. So it will need to be implemented with a method other than the Tailwind `[&:has(svg)]:` syntax because that uses the css `:has()` selector which isn't enabled by default in Firefox.

You should be able to enable the `:has()` selector in you Firefox config. 
https://developer.mozilla.org/en-US/docs/Web/CSS/:has

> From version 103: this feature is behind the layout.css.has-selector.enabled preference (needs to be set to true). To change preferences in Firefox, visit about:config.